module/constants.ts

```diff
@@ -46,6 +46,8 @@ export type SkillTypeString =
     "musical" | "peasant" | "physical" | "schoolofthought" |
     "seafaring" | "special" | "social" | "sorcerous" | "training";
 
+export type FightAttr = "speed" | "agility" | "power" | "skill" | "steel";
```

Comment:
I don't foresee anyone other than the fight dialog and `FightRoll.ts` needing this. I think it's fine to just pull into FightDialog.ts